packages/api/tests/animal_batch.test.js

```diff
@@ -439,4 +462,28 @@ describe('Animal Batch Tests', () => {
       expect(res.status).toBe(400);
     });
   });
+
+  describe('Animal batch model tests', () => {
+    test('Unique internal_identifier should be added within the same farm_id between animal and animal_batch tables', async () => {
+      const [firstFarm] = await mocks.farmFactory();
+      const [secondFarm] = await mocks.farmFactory();
+      const [thirdFarm] = await mocks.farmFactory();
+
+      const testCases = [
+        { farm: firstFarm, isAnimal: false, expectedInternalIdentifier: 1 },
```

Comment:
Just a minor thing, but since the test cases are defined separately from the function call `makeAnimalOrBatchForFarm`, maybe a string format would be more immediately readable than an `isAnimal` boolean?

```js
{ farm: firstFarm, kind: 'animal', expectedInternalIdentifier: 1 },
{ farm: firstFarm, kind: 'batch', expectedInternalIdentifier: 2 },

Reply:
I thought about it, but in that function, I would only do `if (kind === 'animal')`, and I would not use 'batch', so I went with `isAnimal`.
But I don't have a strong preference, so I will update it :)
Thank you for reviewing!

Reply:
You're right, it's a little more weird in the ternary, but I do like it, thank you!! 😂